tools/cmd/sdc-usbkey.js

```diff
@@ -78,7 +85,26 @@ init(opts, args, callback)
             self.uk_ngz = true;
         }
 
-        mod_cmdln.Cmdln.prototype.init.call(self, opts, args, callback);
+        if (!opts.usb) {
+            /*
+             * Unless forced by -u/--usb, see if we booted from a ZFS pool
+             * and set accordingly.
+             */
+            lib_bootpool.triton_bootpool(function (err, pool) {
```

Comment:
I know some of the older code didn't do this, but I think there's a preference to name the callback functions since it can help with debugging (though if Pedro or Todd review any of this, I'll defer to them on it).

Reply:
I'm going to leave this one alone for now.